haiku-apps/wakeup/wakeup-1.0.recipe

```diff
@@ -0,0 +1,45 @@
+
+SUMMARY="An application that plays sound at given interval"
+DESCRIPTION="WakeUp v1.0,\
+Copyright (c)1999. WakeUp is a simple application that will play a given sound at a specified interval.\
+ Compiled under DeOS R4.5.1 (Intel Only)"
+HOMEPAGE="https://github.com/HaikuArchives/WakeUp"
+COPYRIGHT="1999  Jonathan Villemure"
+LICENSE="MIT"
+REVISION="1"
+
+SOURCE_URI="https://github.com/HaikuArchives/WakeUp/archive/116afedea73ba9779d3d641028b65fdb2dc19012.tar.gz"
+CHECKSUM_SHA256="28eb8504a2a436a117821aaa399b8c37f5363bd9fcb2ff814f6cb1b6264da311"
+SOURCE_DIR="WakeUp-116afedea73ba9779d3d641028b65fdb2dc19012"
+
+ARCHITECTURES="x86 x86_gcc2 x86_64"
+
+PROVIDES="
+	wakeup
+	app:WakeUp = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
+{
+	make OBJ_DIR=objects \
+	BUILDHOME=`finddir B_SYSTEM_DEVELOP_DIRECTORY`
```

Comment:
BUILDHOME=`finddir B_SYSTEM_DEVELOP_DIRECTORY` shouldn't be needed with a current Makefile.